doc/usage.md

```diff
@@ -4,10 +4,10 @@ On Linux after building, the following executables are available in the project
 * picohttp_ct: runs various HTTP tests
 * picoquic_ct: runs various QUIC tests
 	* with no arguments, runs all tests.
-* picoquicdemo: QUIC/HTTP demo client and server
+* picoquicdemo: QUIC/HTTP demo client and server for HTTP/3, HTTP/0.9, QUIC performance tests and Siduck(simple ping-pong)
 * picoquic_sample: QUIC/HTTP demo client and server. See sample/README.md for detailed usage
 
```

Comment:
The "sample" code is really just that -- a code sample demonstrating how to write an application using the picoquic stack. It is not meant to be actually used.

Reply:
revised description based on your comment